app/src/main/res/values/strings.xml

```diff
@@ -30,6 +31,9 @@
     <string name="aliases_desc">Enable aliases (convenience shortcuts)</string>
     <string name="aliases_name">Aliases</string>
 
+    <string name="events_desc">Enable calendar events</string>
```

Comment:
Maybe "Search in my calendar events" is better? (i.e. same as "Search in my contacts")